### PR TITLE
Add action masking for VLA0

### PR DESCRIPTION
**Pull request by southfreebird**

### CODE REVIEW

File: src/lerobot/policies/vla0/modeling_vla0.py [hunk 1]
```diff
@@ -134,6 +134,34 @@ def __init__(self, config: VLA0Config):
...
+        self.processor.tokenizer.add_tokens([self.actions_mask_symbol], special_tokens=True)
+        self.vlm.resize_token_embeddings(len(self.processor.tokenizer), mean_resizing=False)
+        self.mask_token_id = self.processor.tokenizer.convert_tokens_to_ids(self.actions_mask_symbol)
+
+    def apply_action_masking(self, actions: list[list[str]]):
+        if not self.training:
+            return actions
+
+        # Skip action masking augmentation for 10% of data
+        if random.random() < 0.1:
```
Comment (balakhnov) [inline src/lerobot/policies/vla0/modeling_vla0.py:149]: Maybe it should be a param?

**Review by balakhnov — APPROVED** [review]
